### PR TITLE
Database migration scripts

### PR DESCRIPTION
**Pull request by shtukas**

### CODE REVIEW

File: flyway-mutation-dev-DB.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+dbname=recipeasy
+dbuser=recipeasy
```
Comment (LATaylor-guardian) [inline flyway-mutation-dev-DB.sh]: This variable is unused so can be deleted.

File: flyway-mutation-prod-DB.sh [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Sets up a tunnel through a recipeasy instance,
+# creates a Postgres user and DB schema,
```
Comment (LATaylor-guardian) [inline flyway-mutation-prod-DB.sh]: Comment needs updating to reflect this script.

**Comment by LATaylor-guardian** [comment]
Other than my comments, nice work. 👍

File: flyway-mutation-dev-DB.sh (hunk 1 above)
Comment (shtukas) [inline flyway-mutation-dev-DB.sh]: Corrected. Thanks!

File: flyway-mutation-prod-DB.sh (hunk 2 above)
Comment (shtukas) [inline flyway-mutation-prod-DB.sh]: Corrected. Thanks!

**Pull request merged**